src/Fetch.js

```diff
@@ -190,7 +190,7 @@ function fetchLoadCachedData(db, fetch, onsuccess, onerror) {
   }
 }
 
-function fetchCacheData(/** @type {IDBDatabase} */ db, fetch, data, onsuccess, onerror) {
+function fetchCacheData(/** @type {IDBDatabase} */ db, fetch, dest, data, onsuccess, onerror) {
```

Comment:
@sbc100 I restored the IndexedDB key to use the request url instead of the response one that was the effect of updating the url field. This was the unit tests failure root cause and now they are passing. Please let me know if this is acceptable approach.